.github/workflows/build-mudlet-win.yml

```diff
@@ -99,6 +102,70 @@ jobs:
       shell: msys2 {0}
       run: $GITHUB_WORKSPACE/CI/package-mudlet-for-windows.sh
 
+    - name: Upload debug symbols to sentry.io (Windows)
+      if: env.WITH_SENTRY == 'yes'
+      shell: msys2 {0}
+      env:
+        SENTRY_AUTH_TOKEN: ${{secrets.SENTRY_AUTH_TOKEN}}
```

Comment:
You won't have access to secrets during a PR build - they'll only be accessible during "Release" and "PTB" type builds.